src/config/backpack/base.php

```diff
@@ -14,6 +14,8 @@
     // Date & Datetime Format Syntax: https://carbon.nesbot.com/docs/#api-localization
     'default_date_format'     => 'D MMM YYYY',
     'default_datetime_format' => 'D MMM YYYY, HH:mm',
+    'default_time_format'     => 'HH:mm',
+    'default_month_format'    => 'M',
```

Comment:
Since these are only used in one place (one column each), let's move them there. No point in adding things to the config file when we can add them to the column file itself.